/motor_control_node:
  ros__parameters:
    /motor2_control_node:
      target_positions: [0.0, 60.0, 0.0, 90.0]
      speed: 10000
      accel: 1000
      min_angle: 0.0
      max_angle: 90.0
      reach_count_max: 100

# ros2 run motor_driver motor_control_node --ros-args --params-file ~/Desktop/AK_ws/src/motor_driver/config/params.yaml